yak/rest_core/models.py

```diff
@@ -46,7 +48,7 @@ def resize_model_photos(obj):
 
     original_file = getattr(obj, original_file_field_name)
     if not original_file:
-        for size_name, size in obj.SIZES.iteritems():
+        for size_name, size in obj.SIZES.items():
```

Comment:
There's another place where `<something>.items()` is coerced into a `list`. Should we be consistent and coerce to list here?